csv/ParameterDefs.csv

```diff
@@ -35,8 +35,8 @@ BASE,,practical_salinity,PD13,function,,float32,,1,-9999999,Practical Salinity,4
-OPTAA,,beam_attenuation,PD19,function,"[""wavelength_c""]",float32,,m-1,-9999999,Optical Beam Attenuation Coefficient,6,,PFID51,"{""cref"":""PD595"",""csig"":""PD597"",""traw"":""PD590"",""cwl"":""CC_cwlngth"",""coff"":""CC_ccwo"",""tcal"":""CC_tcal"",""tbins"":""CC_tbins"",""tc_arr"":""CC_tcarray"",""T"":[""dpi_TEMPWAT_L1"",""dpi_TEMPSRF_L1""],""PS"":[""dpi_PRACSAL_L2"",""dpi_SALSURF_L2""]}",OPTATTN,,OPTATTN_L2,The Optical Beam Attenuation Coefficient is the rate that the intensity of a beam of light will decrease in response to the combined effects of absorption and scatter as a function of propagation distance. The Attenuation Coefficient results from the spectral beam attenuation of the combination of all seawater impurities including all particulate and dissolved matter of optical importance. Units: m-1,Optical Beam Attenuation Coefficient,Science Data,L2
-OPTAA,,optical_absorption,PD20,function,"[""wavelength_a""]",float32,,m-1,-9999999,Optical Absorption Coefficient,6,,PFID52,"{""aref"":""PD596"",""asig"":""PD598"",""traw"":""PD590"",""awl"":""CC_awlngth"",""aoff"":""CC_acwo"",""tcal"":""CC_tcal"",""tbins"":""CC_tbins"",""ta_arr"":""CC_taarray"",""cpd_ts"":""dpi_OPTATTN_L2"",""cwl"":""CC_cwlngth"",""T"":[""dpi_TEMPWAT_L1"",""dpi_TEMPSRF_L1""],""PS"":[""dpi_PRACSAL_L2"",""dpi_SALSURF_L2""]}",OPTABSN,,OPTABSN_L2,Optical Absorption Coefficient is the rate that the intensity of a beam of light will decrease in response to the absorption (removal) of light energy as a function of propagation distance. The Optical Absorption Coefficient reflects the absorption coefficient for the combination of all seawater impurities including all particulate and dissolved matter of optical importance. Units: m-1,Optical Absorption Coefficient,Science Data,L2
+OPTAA,,beam_attenuation,PD19,function,"[""wavelength""]",float32,,m-1,-9999999,Optical Beam Attenuation Coefficient,6,,PFID51,"{""cref"":""PD595"",""csig"":""PD597"",""traw"":""PD590"",""cwl"":""CC_cwlngth"",""coff"":""CC_ccwo"",""tcal"":""CC_tcal"",""tbins"":""CC_tbins"",""tc_arr"":""CC_tcarray"",""T"":[""dpi_TEMPWAT_L1"",""dpi_TEMPSRF_L1""],""PS"":[""dpi_PRACSAL_L2"",""dpi_SALSURF_L2""]}",OPTATTN,,OPTATTN_L2,The Optical Beam Attenuation Coefficient is the rate that the intensity of a beam of light will decrease in response to the combined effects of absorption and scatter as a function of propagation distance. The Attenuation Coefficient results from the spectral beam attenuation of the combination of all seawater impurities including all particulate and dissolved matter of optical importance. Units: m-1,Optical Beam Attenuation Coefficient,Science Data,L2
```

Comment:
What changed here? New line character?

Reply:
per 13025 wavelength dimensions were changed to share a single wavelength dimension